tree/20_cfe_basics/package_lib.cf

```diff
@@ -0,0 +1,38 @@
+#####################################################################################
+# Copyright 2016 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @agent_version >=3.7
+#
+# This collection of bodies configure the package modules
+# As it require the new package modules introduced in 3.7, we need to load it only on CFEngine 3.7.
+
+body package_module apt_get
+{
+    # Query package updates only every 24 hours.
+    query_updates_ifelapsed => "1440";
+    # Query installed packages every 2 hours.
+    query_installed_ifelapsed => "240";
+}
+
+body package_module yum
+{
+    # Query package updates only every 24 hours.
+    query_updates_ifelapsed => "1440";
```

Comment:
i can make a dumb answer: add parameters in the body
a more clever answer would be to have logical default, and be able to define them globally (it doesn't make sense to have different values for different package on a given node). So it may be a generic method "configure package update frequency" that would define it through a bundle common something.
What do you think of it ?